src/NuGet.Clients/VisualStudio.Facade/Events/IRestoreEvents.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.VisualStudio.Facade.Events
+{
+    public delegate void SolutionRestoreCompletedEventHandler(SolutionRestoredEventArgs args);
+
+    /// <summary>
+    /// The interface for listening to events concerning restore operations.
+    /// </summary>
+    public interface IRestoreEvents
+    {
```

Comment:
why we have added these interfaces to NuGet Facade? there is nothing specific to VS14 or VS15 here since these only applicable to VS15. Why don't we add these into NuGet.Tools itself.